packages/hopli/src/identity.rs

```diff
@@ -0,0 +1,80 @@
+use crate::key_pair::create_identity;
+use clap::Parser;
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use crate::utils::{Cmd, HelperErrors};
+
+/// CLI arguments for `hopli identity`
+#[derive(Parser, Default, Debug)]
+pub struct IdentityArgs {
+    #[clap(
+        help = "Password to encrypt identity files",
+        long,
+        short,
+        default_value = ""
+    )]
+    password: String,
+
+    #[clap(
+        help = "Path to the directory that stores identity files",
+        long,
+        short,
+        default_value = "/tmp"
```

Comment:
Should be more specific like `/tmp/hopli`

Reply:
updated